draft-ietf-mls-protocol.md

```diff
@@ -2732,8 +2724,8 @@ New ciphersuite values are assigned by IANA as described in
 
 ## MLS Designated Expert Pool {#de}
 
-[[ OPEN ISSUE: pick DE mailing address.
-Maybe mls-des@ or mls-de-pool. ]]
+<!-- OPEN ISSUE: pick DE mailing address.
```

Comment:
I don't think this quite matches the intent -- we want these OPEN ISSUES to be visible in the rendered HTML

Reply:
I noticed this is how QUIC WG does it: https://github.com/quicwg/base-drafts/commit/09338d0e237e67c648aa172c32b872e10bf737ff